### DISCUSSION TITLE
Remove klayout build step and dependency?

### DISCUSSION THREAD

**Discussion by oharboe** [The-OpenROAD-Project/OpenROAD-flow-scripts#1023] [General] [answered]
I understand that for normal usage, the OpenROAD GUI has taken the place of klayout.

Is there a reason to keep klayout build step as part of the normal flow?

Perhaps klayout can be removed as a default step and removed as a mandatory dependency?

**Comment by rovinski** [comment, marked as answer]
KLayout is still used for GDS generation, which is the endpoint for any tapeout flow. If you want to stop before GDS generation, you can `make $(RESULTS_DIR)/6_final.def` and then KLayout won't be invoked.

**Reply by oharboe** [reply]
Ah, I see. I thought klayout was only a GUI to inspect layout.
